src/components/pages/notifier/mypurchase/mapMyPurchases.tsx

```diff
@@ -7,15 +7,17 @@ import { NotifierSubscriptionItem } from 'models/marketItems/NotifierItem'
 import { ExchangeRate } from 'context/Market/interfaces'
 import MarketplaceAddressCell from 'components/molecules/MarketplaceAddressCell'
 import ExpirationDate, { SubscriptionExpirationType } from 'components/molecules/ExpirationDate'
-import { PlanDTO, PLAN_STATUS } from 'api/rif-marketplace-cache/notifier/offers/models'
 import { SUBSCRIPTION_STATUSES } from 'api/rif-notifier-service/models/subscriptions'
+import { SUBSCRIPTION_STATUS } from 'api/rif-marketplace-cache/notifier/subscriptions/models'
 
 const EXPIRATION_WARNING_TRIGGER = 5
 
 const getExpirationType = (
-  { planStatus, daysLeft }: PlanDTO,
+  { status, expirationDate }: Pick<NotifierSubscriptionItem, 'status' | 'expirationDate'>,
 ): SubscriptionExpirationType => {
-  if (planStatus !== PLAN_STATUS.ACTIVE || daysLeft <= 0) return 'blocked'
+  const daysLeft = Math.ceil(Math.abs(new Date().getTime() - expirationDate?.getTime()) / 86400000)
```

Comment:
```suggestion
  const daysLeft = Math.ceil(Math.abs(new Date().getTime() - expirationDate?.getTime()) / 60*60*24*1000)
```

Or create a util function `timestampToDays(milliseconds)`